examples/light-switch-app/silabs/src/BindingHandler.cpp

```diff
@@ -28,6 +28,9 @@
 
 using namespace chip;
 using namespace chip::app;
+using chip::app::Clusters::LevelControl::MoveModeEnum;
+using chip::app::Clusters::LevelControl::OptionsBitmap;
+using chip::app::Clusters::LevelControl::StepModeEnum;
```

Comment:
We shouldn't use structure / enums like this. We should stop at namespace in this case since this could cause conflicts.
```suggestion
using namespace chip::app::Clusters::LevelControl;
```

Reply:
made this change